pkg/pipeline/transform/netdb/fixtures/etcProtocols.txt

```diff
@@ -0,0 +1,162 @@
+# /etc/protocols:
+# $Id: protocols,v 1.12 2016/07/08 12:27 ovasik Exp $
```

Comment:
As noted in `go help test`, I think `testdata` is more idiomatic name for the fixtures directory.

```
The go tool will ignore a directory named "testdata", making it available
to hold ancillary data needed by the tests.
```
https://pkg.go.dev/cmd/go/internal/test

Reply:
Good point. Didn't know it. Thanks!